stack.yaml

```diff
@@ -0,0 +1,7 @@
+resolver: lts-11.8
```

Comment:
Maybe you could try to use latest resolver from Stackage? I guess we figured out that the problem was not in LTS. So I think you can use newer GHC version. And also you can remove those packages from `extra-deps` that are in the LTS resolver you're going to use.